packages/convai-widget-core/src/widget/LanguageSelect.tsx

```diff
@@ -0,0 +1,70 @@
+import * as Select from "@radix-ui/react-select";
+import { Language } from "@11labs/client";
+import { useState } from "preact/compat";
+import { Icon } from "../components/Icon";
+import { useLanguageConfig } from "../contexts/language-config";
+import { Flag } from "../components/Flag";
+import { useRootPortal } from "../contexts/root-portal";
+import { ComponentChildren } from "preact";
+import { SelectContentProps } from "@radix-ui/react-select";
+
+interface LanguageSelectProps extends SelectContentProps {
+  children: ComponentChildren;
+}
+
+export function LanguageSelect({ children, ...rest }: LanguageSelectProps) {
+  const [open, setOpen] = useState(false);
+  const { language, setLanguage, options } = useLanguageConfig();
+  const portal = useRootPortal();
+
+  return (
+    <Select.Root
+      open={open}
+      value={language.value.languageCode}
+      onValueChange={value => {
+        setLanguage(value as Language);
+        setOpen(false);
+      }}
```

Comment:
Unsure if you tried to go through the flow with screen readers, but since it's going to be a complex form (chat/voice interface) around the web, might be useful. Some blind users would might end up no able to contact support.